.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+    "root": true,
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-eval": "error",
+        "array-bracket-spacing": "error",
+        "no-array-constructor": "error",
+        "block-spacing": "error",
+        "brace-style": "error",
+        "camelcase": "error",
+        "comma-dangle": "error",
+        "comma-spacing": "error",
+        "comma-style": "error",
+        "func-call-spacing": "error",
+        "indent": ["error", 4],
+        "key-spacing": "error",
+        "keyword-spacing": "error",
+        "linebreak-style": ["error", "unix"],
+        "no-multiple-empty-lines": "error",
+        "no-plusplus": "error",
```

Comment:
I find this rule weird but that is probably just a habit so I do not oppose it. In either case `i += 1` would be preferred replacement to `i = i + 1`.